Makefile

```diff
@@ -27,4 +31,18 @@ test:
 test-coverage:
 	py.test --verbose --cov-report term --cov=integrade --cov=tests tests
 
+ci-oc-login:
+	./oc login $(OPENSHIFT_URL) --token=$(OPENSHIFT_TOKEN)
+
+ci-refresh-cloudigrade:
+	REVISION=$$(./oc rollout latest dc/postgresql -o revision)
+	./oc rollout status dc/postgresql --revision=$(REVISION)
+	REVISION=$$(./oc rollout latest dc/cloudigrade -o revision)
+	./oc rollout status dc/cloudigrade --revision=$(REVISION)
+
+ci-create-superuser-and-fetch-token:
+	./oc rsh -c cloudigrade $$(./oc get pods -o jsonpath='{.items[*].metadata.name}' -l name=cloudigrade) scl enable rh-postgresql96 rh-python36 -- python manage.py createsuperuser --no-input --username $(CLOUDIGRADE_USER) --email="$(CLOUDIGRADE_USER)@example.com"
+	CLOUDIGRADE_TOKEN=$$(./oc rsh -c cloudigrade $$(./oc get pods -o jsonpath='{.items[*].metadata.name}' -l name=cloudigrade) scl enable rh-postgresql96 rh-python36 -- python manage.py drf_create_token $(CLOUDIGRADE_USER) | awk '{print $3}')
```

Comment:
can you export just the plain $CLOUDIGRADE_TOKEN as well, from this reading I'm not sure if it is globally available after this command runs